src/main/java/com/cisco/trex/stateful/TRexAstfClient.java

```diff
@@ -368,4 +372,92 @@ public String getVersion() {
         }
     }
 
+    /**
+     * get template group names
+     *
+     * @return template group names
+     */
+    public List<String> getTemplateGroupNames() {
+        return this.getTemplateGroupNames(null);
+    }
+
+    /**
+     * get template group names
+     *
+     * @param profileId
+     * @return template group names
+     */
+    public List<String> getTemplateGroupNames(String profileId) {
+        Map<String, Object> payload = createPayload(profileId);
+        payload.put("epoch", 1);
+        payload.put("initialized", false);
```

Comment:
these two hard-code parameter have different values in example in doc, is it right?

Reply:
at first, I set initialized is true, the same as example. but sometimes it can not get tgNames correctly, so I tried to change to false, It seems always work well. and epoch is the same as example.